src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu.context.tsx

```diff
@@ -65,6 +73,12 @@ export const ResponsiveVerticalMenuProvider = ({
   const [responsiveMode, setResponsiveMode] = useState<boolean>(false);
   const [reducedMotion, setReducedMotion] = useState<boolean>(false);
 
+  const [left, setLeft] = useState("auto");
+  const [top, setTop] = useState("auto");
+
+  const [width, setWidth] = useState<string | undefined>(undefined);
```

Comment:
Comment: I don't think we need state here, we can just pass the props down via the context as it never gets updated internally etc